### DISCUSSION TITLE
ReactMarkDown that uses "children"

### DISCUSSION THREAD

**Discussion by klides** [remarkjs/remark#979] [Q&A] [answered]
How can I resolve ReactMarkDown that uses "children" as a prop w/ Error Do not pass children as props

I'm using next and it gives me this error "Error: Do not pass children as props. Instead, nest children between the opening and closing tags."

**Comment by wooorm** [comment, marked as answer]
- it’s a warning from a tool that checks your code, you can google that tool and find how to hide warnings.
- Then don’t pass it as a prop, but as “children”: `<Markdown>{stuff}<Markdown>`, where `stuff` is a reference to the thing you want to pass.

**Reply by wooorm** [reply]
It should work.
